internal/v4/relations_test.go

```diff
@@ -766,6 +768,53 @@ func (s *RelationsSuite) TestMetaBundlesContaining(c *gc.C) {
 	}
 }
 
+func (s *RelationsSuite) TestMetaBundlesContainingBundleACL(c *gc.C) {
+	// Add the bundles used for testing to the database.
+	for id, b := range metaBundlesContainingBundles {
+		url := mustParseResolvedURL(id)
+		// The blob related info are not used in these tests.
+		// The charm-bundle relations are retrieved from the entities
+		// collection, without accessing the blob store.
+		err := s.store.AddBundle(b, charmstore.AddParams{
+			URL:      url,
+			BlobName: "blobName",
+			BlobHash: fakeBlobHash,
+			BlobSize: fakeBlobSize,
+		})
+		c.Assert(err, gc.IsNil)
+		if url.URL.Name == "useless" {
```

Comment:
Add a comment about removing read permission to everyone for the useless bundle?